Tests/Packet++Test/Tests/NflogTests.cpp

```diff
@@ -29,32 +28,31 @@ PTF_TEST_CASE(NflogPacketParsingTest)
 
 	PTF_ASSERT_EQUAL(nflogLayer->getNextLayer()->getProtocol(), pcpp::IPv4, enum);
 
-	pcpp::NflogTlvType expectedTypes[6] = {
-		pcpp::NflogTlvType::NFULA_PACKET_HDR,
-		pcpp::NflogTlvType::NFULA_PREFIX,
-		pcpp::NflogTlvType::NFULA_IFINDEX_OUTDEV,
-		pcpp::NflogTlvType::NFULA_UID,
-		pcpp::NflogTlvType::NFULA_GID,
-		pcpp::NflogTlvType::NFULA_PAYLOAD
-	};
-
-	int optSizes[6] = {8, 8, 8, 8, 8, 68};
-	std::string optDataAsHexString[6] = {
-		"0800010000000300",
-		"05000a0000000000",
-		"0800050000000002",
-		"08000b0000000000",
-		"08000e0000000000",
-		"410009004500003d021040004011208f0a00020f0a000203a542003500294156c04e0100000100000000000003777777076578616d706c65036e657400000100012f0a31"
-	};
-
-	for (int i = 0; i < 6; i++) {
+	pcpp::NflogTlvType expectedTypes[6] = { pcpp::NflogTlvType::NFULA_PACKET_HDR,
+		                                    pcpp::NflogTlvType::NFULA_PREFIX,
+		                                    pcpp::NflogTlvType::NFULA_IFINDEX_OUTDEV,
+		                                    pcpp::NflogTlvType::NFULA_UID,
+		                                    pcpp::NflogTlvType::NFULA_GID,
+		                                    pcpp::NflogTlvType::NFULA_PAYLOAD };
+
+	int optSizes[6] = { 8, 8, 8, 8, 8, 68 };
+	std::string optDataAsHexString[6] = { "0800010000000300",
+		                                  "05000a0000000000",
+		                                  "0800050000000002",
+		                                  "08000b0000000000",
+		                                  "08000e0000000000",
+		                                  "410009004500003d021040004011208f0a00020f0a000203a542003500294156c04e01000001"
+		                                  "00000000000003777777076578616d706c65036e657400000100012f0a31" };
```

Comment:
This is a bit weird. Maybe we can move it back to one line and ignore clang-format in this case?

Reply:
For this case, I think the formatted code is clean as well. I think it's okay to keep it.

Reply:
maybe we can leave it, but clang-tidy complains about it (I can see it in CLion):
![image](https://github.com/user-attachments/assets/e379df1c-7a59-4a83-9469-b6baabeb5240)

Reply:
I see, let me revert then.